### PR TITLE
feat: Add drizzle sample for javascript

### PR DESCRIPTION
**Pull request by dengfuping**
- Ref: #16

---

# Connect to OceanBase with Drizzle

This document describes how to connect to [OceanBase](https://www.oceanbase.com) with [Drizzle](https://orm.drizzle.team).

## Preparation

Make sure `Node.js` and `npm` are installed.

## Usage

Clone the project and navigate to the appropriate directory:

```bash
git clone git@github.com:oceanbase/ob-samples.git
cd javascript/drizzle
```

Install dependencies:

```bash
npm install
```

Modify the connection string in the `.env` file:

```bash
DATABASE_URL="mysql://root:@127.0.0.1:2881/test"
```

Synchronize the `users` model defined in `db/schema.ts` to the database:

```bash
npx drizzle-kit push
```

Execute `index.ts`:

```bash
npx ts-node index.ts
```

The output should be as follows, indicating successful execution:

```bash
[ { id: 1, email: 'alice@oceanbase.com', name: 'Alice' } ]
```

Check the corresponding `users` table and the data has been inserted:

```bash
mysql> select * from users;
+----+---------------------+-------+
| id | email               | name  |
+----+---------------------+-------+
|  1 | alice@oceanbase.com | Alice |
+----+---------------------+-------+
1 row in set (0.01 sec)
```

### CODE REVIEW

**Comment by whhe** [comment]
If the component can be executed by command line, please add a run.sh script for it to test it on GitHub Action. Otherwise please add the module to applications directory.

**Comment by dengfuping** [comment]
- Related: https://github.com/drizzle-team/drizzle-orm/pull/3721

File: .github/workflows/javascript.yml [hunk 1]
```diff
@@ -17,8 +17,12 @@ jobs:
         module:
           - name: 'mysql2'
             with_oceanbase_container: true
+          - name: 'drizzle'
+            with_oceanbase_container: true
+            oceanbase_image_tag: '4.2.3_BETA'
```
Comment (dengfuping) [inline .github/workflows/javascript.yml:22]: only `oceanbase-ce >= 4.2.3` supports `serial` data type and that is required for Drizzle.

**Review by whhe — APPROVED** [review]
Great work!

**Pull request merged**